### PR TITLE
Add highlighting to company search

### PR DESCRIPTION
**Pull request by richtier**
[this task](https://uktrade.atlassian.net/browse/ED-1957)

image

### CODE REVIEW

File: enrolment/static/sass/company-search_results.scss [hunk 1]
```diff
@@ -561,7 +561,11 @@ h1 {
 }
 
 .ed-company-search-summary {
-  margin-bottom: 10px;
+  margi-bottom: 10px;
```
Comment (konradko) [inline enrolment/static/sass/company-search_results.scss]: typo?

File: company/helpers.py [hunk 2]
```diff
@@ -77,9 +78,19 @@ def get_company_list_from_response(response):
...
+    formatted_results = []
+
+    for result in parsed['hits']['hits']:
+        formatted = format_company_details(result['_source'])
+        if 'highlight' in result:
+            highlighted = '...'.join(
+                result['highlight'].get('description', '') or
+                result['highlight'].get('summary', '')
+            )
+            formatted['highlight'] = mark_safe(highlighted)
```
Comment (konradko) [inline company/helpers.py]: looks like we're marking safe a string that is coming from user input. Won't that expose us to XSS attacks? e.g. what if description contains
```
<style class=class1 onmouseover=javascript:func()>...</style>
```

https://docs.djangoproject.com/en/1.9/topics/security/#cross-site-scripting-xss-protection

File: enrolment/static/sass/company-search_results.scss (hunk 1 above)
Comment (richtier) [inline enrolment/static/sass/company-search_results.scss]: oops. yes. good eye.

File: company/helpers.py (hunk 2 above)
Comment (richtier) [inline company/helpers.py]: ah yes. I will investigate how to allow only the `<em>` tags to be marked as safe

**Pull request merged**